apps/app/src/components/PageComment.tsx

```diff
@@ -84,28 +85,44 @@ export const PageComment: FC<PageCommentProps> = memo((props:PageCommentProps):
   const onDeleteCommentAfterOperation = useCallback(() => {
     onCancelDeleteComment();
     mutate();
-  }, [mutate, onCancelDeleteComment]);
+    if (onCommentCountUpdated != null) {
+      onCommentCountUpdated();
+    }
```

Comment:
I think avoiding null should be done at a higher level component.

If you need this `onCommnetCountUpdated()` process, please make sure that you do not need to pass mutate function as props on to other places that use `PageComment` component.